src/main/java/org/jabref/model/entry/ParsedFileField.java

```diff
@@ -1,7 +1,14 @@
 package org.jabref.model.entry;
 
+import java.io.File;
 import java.net.URL;
+import java.nio.file.Path;
 import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.logic.util.io.FileUtil;
```

Comment:
The thing is, `FileUtil` has more dependencies to logic classes, which would then also need to move to model. At some point, the distinction between the two packages would be completely lost.

Can't you just add a method `public static Path getPath(ParsedFileField fileField)` to `FileUtil` instead? Not much better than the current version, I know, but at least a little. Alternatively, extract the methods you need, and just those, into a new `FileUtil` class in model (maybe with a better name). It wouldn't hurt to break up these util classes into more meaningful smaller units. They all devolve into god classes...